Tita-Clone/Tita-Clone/Controllers/LoginViewController.swift

```diff
@@ -40,9 +149,47 @@ class LoginViewController: UIViewController {
     
     // MARK: - Location
     private func location(){
+        backImg.snp.makeConstraints { make in
+            make.width.height.equalToSuperview()
```

Comment:
```suggestion
            make.edges.equalToSuperview()
```
위와 같은 방법으로도 표현할 수 있습니다. 둘 중 한 가지 방식을 선택해 사용하세요!

Reply:
우와